lib/runner/cli/cli.js

```diff
@@ -109,6 +109,7 @@ class CliRunner {
     this.setupGlobalHooks();
 
     const result = this.readExternalHooks(asyncLoading);
+    this.setMobileOptions(argv);
```

Comment:
we can move this to L:108, since we are setting up logging, and global hooks there, so it make sense to setup Mobile Options as well

Reply:
This PR is still a work in progress as 1 test is failing. I'll turn it into a draft.